firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -369,10 +369,6 @@ class Tab: NSObject, ThemeApplicable, FeatureFlaggable {
             guard nightMode != oldValue else { return }
 
             webView?.evaluateJavascriptInDefaultContentWorld("window.__firefox__.NightMode.setEnabled(\(nightMode))")
-            // For WKWebView background color to take effect, isOpaque must be false,
-            // which is counter-intuitive. Default is true. The color is previously
-            // set to black in the WKWebView init.
-            webView?.isOpaque = !nightMode
```

Comment:
This is outside my wheelhouse, so I'll defer to @issammani 's comment on the PR!